lib/elixometer.ex

```diff
@@ -328,7 +328,7 @@ defmodule Elixometer do
     end
   end
 
-  def time(name, units, elapsed_us) when is_number(elapsed_us) do
+  def update_time(name, units, elapsed_us) when is_number(elapsed_us) do
```

Comment:
This is a breaking change, no?

Reply:
It's actually a new feature. Elixometer currently doesn't support setting a time directly, only by timing a block or function.

Reply:
@andyleclair could you separate this out into it's own pull request? Once that's done, I'll merge this one.

Reply:
@jparise done!